docs/source/examples.rst

```diff
@@ -0,0 +1,16 @@
+Examples
```

Comment:
Please add to CHANGELOG.MD under "Unreleased" "Added" header:
- Extended Theory&Practice RecSys baselines tutorial